### DISCUSSION TITLE
Status of Project?

### DISCUSSION THREAD

**Discussion by woodhull** [tusen-ai/naive-ui#4692] [General]
What's the current status of the naive-ui project? 

I've noticed that the pace of releases and PR reviews/merges have slowed down compared to the previous level of attention from the maintainers. Reading the news I also see that the sponsors of the project TuSimple and Tusen AI have run into legal and financial difficulties.  

I'm committed to the library (we're building a product with it!) but perhaps it might soon be time to explore alternatives for long-term stewardship of the codebase if TuSimple staff do not have capacity to develop it. How's everything going? How can we help as outside contributors interested in the project's success? What resources do the maintainers need?

**Comment by Sepush** [comment]
Maintainance will be continued. Recently very busy at work in recent months and will be back after performance appraisal 🥹

**Comment by jahnli** [comment]
ref  https://github.com/tusen-ai/naive-ui/discussions/5470
